src/backend/access/heap/heapam.c

```diff
@@ -569,7 +569,7 @@ heap_prepare_pagescan(TableScanDesc sscan)
 	/*
 	 * Prune and repair fragmentation for the whole page, if possible.
 	 */
-	heap_page_prune_opt(scan->rs_base.rs_rd, buffer);
+	heap_page_prune_opt(scan->rs_base.rs_rd, buffer, 0);
```

Comment:
I'm confused. Where do you plan to get this info? I must need to look at other commits.